### PR TITLE
dpg, fix properties with specific types not correct in javadoc

### PR DESCRIPTION
**Pull request by haolingdong-msft**
Fix https://github.com/Azure/autorest.java/issues/1666

### CODE REVIEW

File: cadl-tests/.vscode/launch.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
                 "<node_internals>/**"
             ],
             "type": "node"
+        },
```
Comment (haolingdong-msft) [inline cadl-tests/.vscode/launch.json:15]: This is for vs code debugging configuration. With this, you can just click the debug button to debug cadl codegen.

Comment (weidongxu-microsoft) [reply cadl-tests/.vscode/launch.json:15]: so far it is fine.

we might see later if `"runtimeArgs": ["cadl", "compile", "builtin.cadl"],` will affect other usages.

**Review by weidongxu-microsoft — APPROVED** [review]

File: cadl-tests/.vscode/launch.json (hunk 1 above)
Comment (haolingdong-msft) [reply cadl-tests/.vscode/launch.json:15]: You can modify this `runtimeArgs` to debug other cadl files generation

Comment (weidongxu-microsoft) [reply cadl-tests/.vscode/launch.json:15]: yes, but I don't want to either commit it every time it get modified locally, or take the pain to exclude it from commit.

we might later see if put it to ignore.

**Pull request merged**